cmd/backup-operator/README.md

```diff
@@ -0,0 +1,83 @@
+# etcd backup operator
+
+## Overview
+
+etcd backup operator backups the data of a etcd cluster running on [Kubernetes][Kube] to a remote storage such as AWS [S3][s3].
+
+## Getting Started
+
+Try out etcd backup operator by running it on Kubernetes and then create a `EtcdBackup` Custom Resource which contains the targeting etcd cluster and S3 backup config; the etcd backup operator automatically picks up the `EtcdBackup` Custom Resource, retrieves etcd snapshot, and then saves it to S3.
+>Note: The demo uses the `default` namespace.
+
+Prerequisites: 
+* access to a Kubernetes environment.
+* a running etcd cluster named `example-etcd-cluster`. See [Instruction][etcd_cluster_deploy].
```

Comment:
See instructions to deploy etcd cluster.

Reply:
done